### PR TITLE
[script.pystone.benchmark] 1.1.2

### PR DESCRIPTION
**Pull request by Lunatixz**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.pystone.benchmark/resources/lib/pystone.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+        NextRecord.IntComp = Proc7(NextRecord.IntComp, 10)
+    else:
+        PtrParIn = NextRecord.copy()
+    NextRecord.PtrComp = None
+    return PtrParIn
+
+
+def Proc2(IntParIO):
+    IntLoc = IntParIO + 10
+    while 1:
```
Comment (pkscout) [inline script.pystone.benchmark/resources/lib/pystone.py:150]: Running loops like this should, if possible, check for an abort request from Kodi using `xbmc.Monitor().abortRequested()`.

File: script.pystone.benchmark/resources/lib/pystone.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+    CharLoc2 = CharLoc1
+    if CharLoc2 != CharPar2:
+        return Ident1
+    else:
+        return Ident2
+
+
+def Func2(StrParI1, StrParI2):
+    IntLoc = 1
+    while IntLoc <= 1:
```
Comment (pkscout) [inline script.pystone.benchmark/resources/lib/pystone.py:238]: If this is likely to be a long running loop, checking for abort request would be useful here too.

**Review by pkscout — CHANGES_REQUESTED** [review]

File: script.pystone.benchmark/resources/lib/pystone.py (hunk 1 above)
Comment (Lunatixz) [reply script.pystone.benchmark/resources/lib/pystone.py:150]: Correct, however, pystones is an established benchmark;

I would not feel comfortable changing pystone internals it must be held to the same standards (code) as every other device executing it.

IntLoc governs the length the loop; no chance for long instances, msecs to execute. I believe it was found to be 'ok' when I submitted to the Krypton branch several years ago.

Comment (pkscout) [reply script.pystone.benchmark/resources/lib/pystone.py:150]: Got it.  I didn't realize this was, in essence, a module.  I agree, we shouldn't change the internals.

**Review by pkscout — APPROVED** [review]

**Pull request merged**